src/api/server.rs

```diff
@@ -629,22 +629,26 @@ mod tests {
     use super::ServerClient;
     use crate::{api::check::Request, error::Error};
 
-    fn dbg_err(e: &Error) {
-        eprintln!("Error: {e:?}")
+    fn get_testing_server_client() -> ServerClient {
+        ServerClient::new("http://localhost", "8010")
     }
```

Comment:
Good idea to put this under a function. Wouldn't it be even better to use [`#[fixture]`](https://docs.rs/rstest/latest/rstest/attr.fixture.html)?

Reply:
You want me to introduce `rstest`?

Reply:
I thought it was already a dev-dep of this package, but it is not, so let's not add a dev dep just for that!